Algorithms/Dynamic-Programming/LCS.cpp

```diff
@@ -0,0 +1,93 @@
+// Program to find Longest Common Subsequence between two strings
+// using Dynamic Programming Approach.
+
+// Author: Pritam Negi.
+
+#include <iostream>
+#include <string>
+#include <queue>
```

Comment:
Unused in the program.

Reply:
queue header file is declared for vector implementation to store the LCS string

Reply:
But, for that we need to use `#include <vector>`.

Reply:
Yaa csn be done that way also